.travis.yml

```diff
@@ -3,31 +3,31 @@ language: python
 sudo: false
 
 matrix:
-    include:
-        - python: 2.7
-          env: TOXENV=py27
-        - python: 3.3
-          env: TOXENV=py34
-        - python: 3.4
-          env: TOXENV=py34
-        - python: 3.5
-          env: TOXENV=py35
-        - python: pypy
-          env: TOXENV=pypy
-        - python: 3.5
-          env: TOXENV=py2-cover,py3-cover,coverage
-        - python: 3.5
-          env: TOXENV=docs
-        - python: 2.7
-          env: TOXENV=py27-pyramid12
-        - python: 2.7
-          env: TOXENV=py27-pyramid13
-        - python: 2.7
-          env: TOXENV=py27-pyramid14
-        - python: 2.7
-          env: TOXENV=py27-pyramid15
-        - python: 2.7
-          env: TOXENV=py27-pyramid16
+  include:
+    - python: 2.7
+      env: TOXENV=py27
+    - python: 3.4
+      env: TOXENV=py34
+    - python: 3.5
+      env: TOXENV=py35
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+      dist: xenial
+      sudo: true
+    - python: 3.8-dev
+      env: TOXENV=py38
+      dist: xenial
+      sudo: true
```

Comment:
This one should probably be marked to allow failures, given that there is no 3.8 pre-release available.

Reply:
Is this what you mean? https://github.com/Pylons/pyramid_mailer/pull/89/commits/8079f3e78e80c35a0a4f0a100459133ca2c3ff00#diff-354f30a63fb0907d4ad57269548329e3R29